AlfrescoApp/View Controllers/Sync View Controller/Data Source/RepositoryCollectionViewDataSource.m

```diff
@@ -686,4 +686,10 @@ - (void)sessionRefreshed:(NSNotification *)notification
     self.session = notification.object;
 }
 
+- (NSString*)getSearchType
+{
+   return [self getSearchType];
```

Comment:
This would cause an infinite loop, because you are calling yourself. Should really return a default value.